### PR TITLE
Small cleanup in api and tests

### PR DESCRIPTION
**Pull request by LefterisJP**

### CODE REVIEW

**Comment by LefterisJP** [comment]
@hackaugusto I agree and since I was cleaning up anyway I made the necessary changes.

File: raiden/tests/api/test_api.py [hunk 1]
```diff
@@ -592,7 +623,9 @@ def test_break_blockchain_events(
     # the provided token_address
     api_test_context.specify_token_for_channelnew('0x61c808d82a3ac53231750dadc13c777b59310bd9')
     request = grequests.get(
-        'http://localhost:{port}/api/1/events/tokens/0x61c808d82a3ac53231750dadc13c777b59310bd9?from_block=5&to_block=20&token_address=0x167a9333bf582556f35bd4d16a7e80e191aa6476'.format(
+        'http://localhost:{port}/api/1/events/tokens/0x61c808d82a3ac53231750dadc13c777b59310bd9'
```
Comment (hackaugusto) [inline raiden/tests/api/test_api.py]: What about these urls? Could you make use of url_for here too?

File: raiden/tests/api/test_api.py [hunk 2]
```diff
@@ -549,8 +576,12 @@ def test_query_blockchain_events(
     # of `get_channel_events()` but just makes sure the proper data make it there
     api_test_context.specify_channel_for_events('0xedbaf3c5100302dcdda53269322f3730b1f0416d')
     request = grequests.get(
-        'http://localhost:{port}/api/1/events/channels/0xedbaf3c5100302dcdda53269322f3730b1f0416d?from_block=10&to_block=90'.format(
-            port=rest_api_port_number,
+        api_url_for(
+            api_backend,
+            'channeleventsresource',
```
Comment (hackaugusto) [inline raiden/tests/api/test_api.py]: Could you please use a more readable name?

File: raiden/tests/api/test_api.py (hunk 1 above)
Comment (LefterisJP) [inline raiden/tests/api/test_api.py]: Don't know how to provide an argument both times through `url_for()`. The test is to see if we can overwrite the arguments.

File: raiden/tests/api/test_api.py (hunk 2 above)
Comment (LefterisJP) [inline raiden/tests/api/test_api.py]: It needs to be in lowercase. `url_for` works by lowercasing the resource class name which in this case is `ChannelEventsResource`

File: raiden/tests/api/test_api.py (hunk 1 above)
Comment (hackaugusto) [inline raiden/tests/api/test_api.py]: Humm, you could use `api_url_for` to generate the proper URL and the append the qs parameter at the end

**Pull request merged**